### ISSUE TITLE
How to integrate this lib to obj c project?

### ISSUE THREAD

**Issue by karmamaster** [xuan32546/IOS13-SimulateTouch#38]
Hello,
Because of the hid-support library is not support on ios 13-14 anymore! so I was trying with this greate IOS13-SimulateTouch library, But I don't know how to integrate or call it in the obj c. Can you help me? Thank in advance.

**Comment by xuan32546** [comment]
Are you using jailbroken device?

**Comment by karmamaster** [comment]
> 
> 
> Are you using jailbroken device?

yes I am, I am using jailbroek device on 12.5.1 on iphone 5s and 14.4.1 on iphone 6s. I need to call this lib to run like I used to with hid-support lib.
